indico/modules/events/contributions/contrib_fields.py

```diff
@@ -37,15 +38,22 @@ class ContribFieldConfigForm(IndicoForm):
     title = StringField(_('Title'), [DataRequired()], description=_("The title of the field"))
     description = TextAreaField(_('Description'), description=_("The description of the field"))
     is_required = BooleanField(_('Required'), widget=SwitchWidget(),
-                               description=_("Whether the user has to fill out the field"))
+                               description=_("Whether the field has to be filled out"))
     is_active = BooleanField(_('Active'), widget=SwitchWidget(),
                              description=_("Whether the field is available."),
                              default=True)
+    visibility = IndicoEnumRadioField(_('Visibility'), [DataRequired()], default=ContributionFieldVisibility.public,
+                                      enum=ContributionFieldVisibility,
+                                      description=_('Who will be able to see the field'))
+    user_editable = BooleanField(_('User editable'), widget=SwitchWidget(),
+                                 description=_("Whether the submitter/author can fill out the field during abstract"
+                                               " submission."),
```

Comment:
usually we put a space at the end of a previous line rather than at the beginning of the next